app/js/components/manageApps/AddonList.jsx

```diff
@@ -20,6 +20,7 @@ export const AddonList = ({
   updatesAvailable,
   searchedAddons,
   getInstalled,
+  searchAddonsStatus
```

Comment:
Why introduce a new item in state rather than just check if `searchedAddons.length > 0` ?